### DISCUSSION TITLE
Genes and their evolutionary background that are involved in diseases

### DISCUSSION THREAD

**Discussion by i2kmt** [VerisimilitudeX/DNAnalyzer#377] [General]
After [this](https://github.com/VerisimilitudeX/DNAnalyzer/discussions/246#discussioncomment-4660629) issue and the discussion followed with @LimesKey, I forked the repository and worked on finding mutations on genes with an evolutionary background. I created the md [genes-i2kmt](https://github.com/i2kmt/DNAnalyzer/blob/df39be6e92c6d3110139aa04b4ecaf8d2f679fdc/genes-i2kmt.md) to note the information there directly. Please let me know if everything seems right or note the things would like me to change.

**Comment by VerisimilitudeX** [comment]
Hey @i2kmt, thanks for the list! We're actively working on creating a database of genomes and other metadata, so we appreciate your contribution to that. Could you please create a pull request with the list? Thanks!
